<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lc3</class>
 <widget class="QMainWindow" name="lc3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1305</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>lc3</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ASSEMBLE">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ASSEMBLE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RUN">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>540</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Next Phase</string>
    </property>
   </widget>
   <widget class="QLabel" name="AssembleResLable">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>590</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Asseble result:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Phase_lable">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Phase</string>
    </property>
   </widget>
   <widget class="QLabel" name="Phase">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>501</x>
      <y>91</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="R0">
       <property name="text">
        <string>R0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="R0_in">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>501</x>
      <y>448</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="Negative">
       <property name="text">
        <string>Negative</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Negative_in">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>501</x>
      <y>121</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="R1">
       <property name="text">
        <string>R1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="R1_in">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>501</x>
      <y>150</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="R2">
       <property name="text">
        <string>R2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="R2_in">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>501</x>
      <y>180</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="R3">
       <property name="text">
        <string>R3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="R3_in">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>501</x>
      <y>210</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="R4">
       <property name="text">
        <string>R4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="R4_in">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>501</x>
      <y>240</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="R5">
       <property name="text">
        <string>R5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="R5_in">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>501</x>
      <y>269</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="R6">
       <property name="text">
        <string>R6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="R6_in">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>501</x>
      <y>299</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="R7">
       <property name="text">
        <string>R7</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="R7_in">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>501</x>
      <y>329</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="PC">
       <property name="text">
        <string>PC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="PC_in">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>501</x>
      <y>359</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="IR">
       <property name="text">
        <string>IR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="IR_in">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>501</x>
      <y>388</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QLabel" name="MDR">
       <property name="text">
        <string>MDR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="MDR_in">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>501</x>
      <y>418</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QLabel" name="MAR">
       <property name="text">
        <string>MAR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="MAR_in">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>501</x>
      <y>477</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QLabel" name="Zero">
       <property name="text">
        <string>Zero</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Zero_in">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>501</x>
      <y>507</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QLabel" name="Positive">
       <property name="text">
        <string>Positive</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Positive_in">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Upload_code">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>68</y>
      <width>411</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload Your Code here:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="memoryTable">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>20</y>
      <width>341</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1305</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
